--- autofs-5.0.9/Makefile.conf.in.orig	2014-04-28 10:30:33.308045195 +0300
+++ autofs-5.0.9/Makefile.conf.in	2014-04-28 10:32:33.407843164 +0300
@@ -107,6 +107,9 @@
 # Location for init.d files
 initdir = @initdir@
 
+# Location for init.d files configuration
+initconfdir = @initconfdir@
+
 # Location of systemd unit files
 systemddir = @systemddir@
 
--- autofs-5.0.9/aclocal.m4.orig	2014-03-28 07:54:42.000000000 +0300
+++ autofs-5.0.9/aclocal.m4	2014-04-28 10:30:33.307045205 +0300
@@ -97,7 +97,7 @@
 dnl --------------------------------------------------------------------------
 dnl AF_INIT_D
 dnl
-dnl Check the location of the init.d directory
+dnl Check the location of the service script directory
 dnl --------------------------------------------------------------------------
 AC_DEFUN(AF_INIT_D,
 [if test -z "$initdir"; then
@@ -110,6 +110,24 @@
       fi
     fi
   done
+fi])
+
+dnl --------------------------------------------------------------------------
+dnl AF_INITCONF_D
+dnl
+dnl Check the location of the service configuration directory
+dnl --------------------------------------------------------------------------
+AC_DEFUN(AF_INITCONF_D,
+[if test -z "$initconfdir"; then
+  AC_MSG_CHECKING([location of the init.d configuration directory])
+  for initconf_d in /etc/sysconfig /etc/default; do
+    if test -z "$initconfdir"; then
+      if test -d "$initconf_d"; then
+	initconfdir="$initconf_d"
+	AC_MSG_RESULT($initconfdir)
+      fi
+    fi
+  done
 fi])
 
 dnl --------------------------------------------------------------------------
