--- autofs-5.1.1/samples/autofs.service.in.nss_lookup	2015-06-22 12:16:23.554672662 +0300
+++ autofs-5.1.1/samples/autofs.service.in	2015-06-22 12:19:00.074227025 +0300
@@ -1,6 +1,6 @@
 [Unit]
 Description=Automounts filesystems on demand
-After=network.target ypbind.service sssd.service network-online.target
+After=network.target ypbind.service nss-lookup.target sssd.service network-online.target
 Wants=network-online.target
 
 [Service]
